src/main/java/mate/academy/service/UserService.java

```diff
@@ -8,13 +8,12 @@ public class UserService {
             new User("alice@i.ua", "1234")
     };
 
-    /**
-     * Find user by email. All users are stored in <code>private static final User[] users</code>
-     * @param email - the input parameter
-     * @return - user if his email is equal to passed email.
-     * Return <code>null</code> if there is no suitable user
-     */
     public User findByEmail(String email) {
+        for (User userUnit : users) {
```

Comment:
```suggestion
        for (User user : users) {
```
let's call this variable user
try to use simple names when it's obvious what's happening